### PR TITLE
typos and links on take action page

### PR DESCRIPTION
**Pull request by derekeder**
## Overview

- fixes a few typos on the take action page
- adds links for more info on heat pumps and induction stoves (same as state detail page)

### Demo

Screen Shot 2022-07-15 at 4 59 02 PM

## Testing Instructions

* read through copy on take action and confirm changes

### CODE REVIEW

File: src/pages/take-action.js [hunk 1]
```diff
@@ -21,13 +21,13 @@ const TakeActionPage = () => {
           <div className="col-lg-9">
             <h2>Get Personal: Electrify your machines!</h2>
             <p>
-              For most Americans, more than half your carbon footprint comes from how you heat and power your home, and from the car you drive.
+              For most Americans, more than half your carbon footprint comes from how you heat and power your home and from the car you drive.
               More than cutting out meat or single use plastics, the most impactful thing you can do – by far – is to eliminate your personal burning of fossil fuels:
             </p>
             <ol>
-              <li>Replace your heating system with a heat pump, your gas stove with an induction stove, and your water heater with an electric one</li>
+              <li>Replace your heating system with an <a href="https://en.wikipedia.org/wiki/Heat_pump" target="_blank" rel="noreferrer">electric heat pump</a>, your gas stove with an <a href="https://en.wikipedia.org/wiki/Induction_cooking" target="_blank" rel="noreferrer">induction stove</a>, and your water heater with an electric one</li>
```
Comment (vkoves) [inline src/pages/take-action.js:28]: Wouldn't it still be "a heat pump"? (you changed it to "an")

**Review by vkoves — APPROVED** [review]
Looks great to me! We may want to break up long lines of text, it makes the code really hard to follow (though it keeps diffs simpler). The common limit is 80 characters but at my work we do a goal of 100 with an absolute bound of 120 to be a little friendlier.

**Comment by derekeder** [comment]
thx! for line lengths, if we want to follow a convention, i'd prefer to set it in the eslint config
